pkg/networkmanager/interfaces.js

```diff
@@ -1397,7 +1397,7 @@
             return device.activate_with_settings(settings);
         } else {
             cockpit.warn("No way to apply settings", connection, settings);
-            return cockpit.resolve();
+            return Promise.resolve();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)